src/CP/core/cpmodification.jl

```diff
@@ -19,7 +19,7 @@ function updateChildren!(prunedDomains::CPModification, x::Union{IntVar, BoolVar
     for y in x.children
         childrenPruned = childrenValue.([y], pruned)
         if haskey(prunedDomains, y.id)
-            prunedDomains[y.id] = vcat(prunedDomains[y.id], Array(childrenPruned))
+            prunedDomains[y.id] = append!(prunedDomains[y.id], Array(childrenPruned))
```

Comment:
I didn't know that _vcat_ what not the best practice in this case. 

Just to know, what is the core difference between both methods ?

Reply:
The performance difference is small but `vcat` creates a new array, `append!` modifies the array. So in this case the best practice is append! (see https://discourse.julialang.org/t/vcat-vs-push-which-one-is-more-efficient/48353). But you make me realize that the left side of the equal is not necessary, I will change that.